test/components/Modals/AddContactModal.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, getByRole as testGetByRole } from '@testing-library/react';
```

Comment:
I _think_ the CI/CD test failures are caused by this `getByRole` import. That appears to be a function from the [DOM testing library](https://testing-library.com/docs/dom-testing-library/intro), which forms the basis for a lot of the framework specific libraries. I don't think this function is playing nicely with the React Version, [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). The react library handles rendering differently than DOMTL, and I don't think these methods can be mixed and matched. (This is probably actually a bug in the framework itself. If we can get a minimally reproducible example, it may be worth reporting.)

I think if you change from importing `getByRole` to using the getByRole function on either the screen or the renderer, things should start behaving more consistently.

`screen.getByRole('combobox')`

or preferably:
```javascript
{getByRole} = <ReactComponent />
const elt = getByRole('combobox');
```